docs/wiki/End-user-Guide/installation/install-client.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Client
+---
+
+# Client
+
+## How to Install Cleanroom Loader on the Client
+
+### About Launchers
+
+The recommended way to run Cleanroom Loader is with a **MultiMC-based launcher**, such as **Prism Launcher** or **PolyMC**.  
+These are officially supported because other launchers may not properly handle the removal of certain vanilla libraries.
+
+---
+
+### Manual Install
+
+> **Note:** Manual installation of Cleanroom Loader is only supported on MultiMC-based launchers.
```

Comment:
It's not that we don't support vanilla launchers but the installation may be a lot less straightforward, but it uses the installer (same one as the server installer).